Distributor/src/CoverageAggregator.js

```diff
@@ -55,19 +63,29 @@ class Coverage {
 		}
 	}
 
-	final() {
-		let results = [];
+    final() {
+        let results = [];
 
-		for (let fidx in this._current) {
-			let file = this._getFile(fidx);
-			results.push({
-				file: fidx,
-				data: this._results(file)
-			});
-		}
+        for (let fidx in this._current) {
+            let file = this._getFile(fidx);
+            results.push({
+                file: fidx,
+                data: this._results(file),
+                coveredLines: Array.from(file.touchedLines)
+            });
+        }
 
-		return results;
-	}
+        return results;
+    }
+
+    getTouchedLines() {
+        let toStringify = {};
+        Object.keys(this._current).forEach( key => {
```

Comment:
Why object.keys, forEach gives you the index as the second argument this._current.forEach((item, index) => ...)